### ISSUE TITLE
Translation using Poedit

### ISSUE THREAD

**Issue by Mecanik** [HardeepAsrani/my-awesome-plugin#21]
Hello,

Could you please explain how to use poeditor and translate the index.js? I noticed after compilation/minification the __() are removed completely.

I`m not sure how to do this?

Many thanks!

**Comment by HardeepAsrani** [comment]
Hey @Mecanik,

It's an old tutorial so we use a custom field that strips the __ tags. You can update to using @wordpress/scripts instead: https://developer.wordpress.org/block-editor/reference-guides/packages/packages-scripts/

This way you don't have to manage your build and the whole thing become a lot easier.

**Comment by Mecanik** [comment]
Thanks, well I need to learn more about this.

**Comment by HardeepAsrani** [comment]
It's actually pretty easier, way easier than doing a custom build. 😄

**Comment by Mecanik** [comment]
Yeah well... when you have no idea about WP JS "scripts", no patience, no time and you don't like WP in general... 😄
